src/compas_rhino/artists/artist.py

```diff
@@ -371,4 +420,37 @@ def draw_circles(self, circles, layer=None, clear_layer=False, redraw=False):
 
 if __name__ == "__main__":
 
-    pass
+    from compas_rhino.utilities import unload_modules
+    unload_modules("compas")
+    unload_modules("compas_rhino")
```

Comment:
this line is redundant, because the one above (`unload_modules("compas")`) unloads all modules that start with `"compas"`

Reply:
i see, thx for hint!